src/config/select.js

```diff
@@ -1,4 +1,3 @@
-import React from 'react';
```

Comment:
Still, I'm all in on importing React.PropTypes directly. Makes the code easier to read.